app/components/DocumentTableList/DocumentTableList.jsx

```diff
@@ -120,8 +120,8 @@ export default class DocumentTableList extends React.Component {
   }
 
   handleTableSort(value, sortBy, sortOrder) {
-    const {onBuildBaseUrl} = this.props
-    const link = onBuildBaseUrl({
+    const {buildSortUrl = this.props.onBuildBaseUrl} = this.props
```

Comment:
Can you clarify this part? Is this so that sorting will use `buildSortUrl` and fall back to `onBuildBaseUrl`?

Reply:
Yes, exactly